focus/internals/src/lib/model/selection/selection.rs

```diff
@@ -155,7 +156,7 @@ impl<'processor> SelectionOperationProcessor<'processor> {
                         }
                     }
                 }
-                (OperationAction::Remove, Underlying::Target(target)) => {
+                (OperationAction::Remove(_), Underlying::Target(target)) => {
```

Comment:
This reveals the place where the removal actually happens (mentioned in one of your comments) Nice.

Reply:
Great, glad you like it!